### PR TITLE
Update addon resizer in cluster-large.md

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref: #27409

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm
/sig autoscaling

File: content/en/docs/setup/best-practices/cluster-large.md [hunk 1]
```diff
@@ -124,3 +124,6 @@ components, including cluster-critical addons.
 The [cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#readme)
 integrates with a number of cloud providers to help you run the right number of
 nodes for the level of resource demand in your cluster.
+
+The [addon resizer](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#readme)
```
Comment (tengqm) [inline content/en/docs/setup/best-practices/cluster-large.md:128]: The link target can be "https://github.com/kubernetes/autoscaler/blob/master/addon-resizer/README.md" or "https://github.com/kubernetes/autoscaler/blob/master/addon-resizer/", I don't understand what the current value means.

Comment (sftim) [reply content/en/docs/setup/best-practices/cluster-large.md:128]: https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#readme jumps past the GitHub page pager to the README itself.

Comment (sftim) [reply content/en/docs/setup/best-practices/cluster-large.md:128]: Example of how the user agent viewport looks if you visit that hyperlink
example rendering

**Pull request merged**